scripts/Inelastic/IndirectBayes.py

```diff
@@ -13,20 +13,9 @@
 Output : the Fortran numpy array is sliced to Python length using dataY = yout[:ny]
 """
 
-from IndirectImport import is_supported_f2py_platform, import_f2py, unsupported_message
```

Comment:
I think we want to keep lines 16 to 25. Mantid should still be able to run https://docs.mantidproject.org/nightly/algorithms/BayesQuasi-v1.html and https://docs.mantidproject.org/nightly/algorithms/BayesStretch-v1.html

However, they now fail

Reply:
actually the error might be on my end. Will double check

Reply:
the error was me